xbmc/GUIPassword.h

```diff
@@ -64,23 +65,31 @@ class CGUIPassword : public ISettingCallback
   void LockSources(bool lock);
   void RemoveSourceLocks();
   bool IsDatabasePathUnlocked(const std::string& strPath, VECSOURCES& vecSources);
-  /*! \brief Tests if the user is allowed to access the path by looking up the matching Mediasource
-   \param strPath The folder path to access
-   \param strType The type of share being accessed, e.g. "music", "video", etc. See CSettings::UpdateSources()
-   \return If access is granted, returns \e true
-   */
-  bool IsMediaPathUnlocked(const std::string& strPath, const std::string& strType);
+
+  // getter
+  const std::string& GetMediasourcePath() const { return strMediasourcePath; }
+
+  // setter
+  void SetMediasourcePath(const std::string& tmpMediasourcePath)
```

Comment:
Maybe just `mediasourcePath` instead of `tmpMediaSourcePath`

Reply:
makes sense